src/Tribe/Template.php

```diff
@@ -767,6 +760,36 @@ public function template( $name, $context = [], $echo = true ) {
 		return $html;
 	}
 
+	/**
+	 * Includes a give PHP inside of a safe context.
+	 *
+	 * This method is required to prevent template files messing with local variables used inside of the
+	 * `self::template` method. Also shelters the template loading from any possible variables that could
+	 * be overwritten by the context.
+	 *
+	 * @since 4.11.0
+	 *
+	 * @param string $file Which file will be included with safe context.
+	 *
+	 * @return void
+	 */
+	public function template_safe_include( $file ) {
+		// We use this instance variable to prevent collisions.
+		$this->template_current_file_path = $file;
+		unset( $file );
+
+		// Only do this if really needed (by default it wont).
+		if ( true === $this->template_context_extract && ! empty( $this->context ) ) {
+			// Make any provided variables available in the template variable scope.
+			extract( $this->context ); // @phpcs:ignore
+		}
+
+		include $this->template_current_file_path;
+
+		// After the include we reset the variable.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 93921989182a1b66cd9aebb105b38b4d</sub>